### DISCUSSION TITLE
what is nestjs?

### DISCUSSION THREAD

**Discussion by lim950808** [lim950808/JooMaDeung#10] [Q&A] [answered]
thank you

**Comment by jaehafe** [comment, marked as answer]
Hi, I'm happy to share my ideas.

Next.js is a React framework that gives you building blocks to create web applications.

By framework, we mean Next.js handles the tooling and configuration needed for React, and provides additional structure, features, and optimizations for your application

Please refer to this [link ](https://nextjs.org/learn/foundations/about-nextjs/what-is-nextjs)for more Information.

Bests
Adam

**Reply by lim950808** [reply]
Thank you so much
